### PR TITLE
fixed Illustration for Notice board when no notices available 

### PR DESCRIPTION
**Pull request by nikhilagastya**
Added a no notices available component which also redirects you to the home page 

- fixes: #3957

### CODE REVIEW

**Review by rithviknishad — CHANGES_REQUESTED** [review]
You are introducing so many packages and also new material UI packages into the codebase. However, we are trying to get rid of material UI libraries entirely.

Try to achieve the same without using any new packages and use tailwind only.

Also, link issues to your PRs so that it's easier to track. Refer: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by nikhilagastya** [comment]
> You are introducing so many packages and also new material UI packages into the codebase. However, we are trying to get rid of material UI libraries entirely.
> 
> Try to achieve the same without using any new packages and use tailwind only.
> 
> Also, link issues to your PRs so that it's easier to track. Refer: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

alright I thought since material ui is already being used ,why not use it ,i will try to make a custom component with tailwind css
and about the packages I thought they would be helpful for a few features but didnt use them.

**Comment by nikhilagastya** [comment]
@rithviknishad could you check this PR once!
Thankyou

File: src/Components/Notifications/Notificationbell.png
Comment (rithviknishad) [inline src/Components/Notifications/Notificationbell.png:1]: @nikhilagastya keep this file in `public/images` directory along with other images.

File: src/Components/Notifications/NoticeBoard.tsx [hunk 1]
```diff
@@ -55,13 +57,30 @@ export const NoticeBoard: any = () => {
     );
   } else {
     notices = (
-      <Card key="no-notice" className="my-4 rounded-lg">
-        <CardContent>
-          <div className="text-xl text-center semibold">
-            {t("no_notices_for_you")}
+      <div
+        className="max-w-sm rounded overflow-hidden shadow-lg "
+        style={{ marginLeft: "35%", marginTop: "45px" }}
```
Comment (rithviknishad) [inline src/Components/Notifications/NoticeBoard.tsx]: Why do we have both CSS and tailwind here? Use tailwind classes only.

File: package.json
Comment (rithviknishad) [inline package.json:1]: Why are the new packages still present? Remove all of them.

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: Make sure you don't track your `package-lock.json` file as this PR does not have any package changes.

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Notifications/NoticeBoard.tsx [hunk 2]
```diff
@@ -55,13 +57,30 @@ export const NoticeBoard: any = () => {
...
+          </p>
+        </div>
+        <div className="px-6 pt-4 pb-2">
+          <button
+            className="bg-green-500 text-white hover:bg-green-700 text-white font-bold py-2 px-4 rounded-full"
+            style={{ marginLeft: "35%", color: "inherit" }}
+          >
+            <a href="/" style={{ color: "white" }}>
+              Go Home
+            </a>
```
Comment (rithviknishad) [inline src/Components/Notifications/NoticeBoard.tsx]: Use `Link` component. Avoid using anchor tags directly. Again, no CSS if the same can be achieved with tailwind.

File: src/Components/Notifications/NoticeBoard.tsx [hunk 3]
```diff
@@ -55,13 +57,30 @@ export const NoticeBoard: any = () => {
...
           </div>
-        </CardContent>
-      </Card>
+          <p className="text-gray-700 text-base">
+            Did you know: We could render a fun fact about health here
+          </p>
+        </div>
+        <div className="px-6 pt-4 pb-2">
+          <button
+            className="bg-green-500 text-white hover:bg-green-700 text-white font-bold py-2 px-4 rounded-full"
```
Comment (rithviknishad) [inline src/Components/Notifications/NoticeBoard.tsx]: Replace `green` with `primary`.
```suggestion
            className="bg-primary-500 text-white hover:bg-primary-700 text-white font-bold py-2 px-4 rounded-full"
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Notifications/Notificationbell.png
Comment (nikhilagastya) [reply src/Components/Notifications/Notificationbell.png:1]: okay @rithviknishad

File: src/Components/Notifications/NoticeBoard.tsx (hunk 2 above)
Comment (nikhilagastya) [reply src/Components/Notifications/NoticeBoard.tsx]: using navigate would be a better option here ig instead if anchor tag

File: package.json
Comment (nikhilagastya) [reply package.json:1]: can you tell me which all packages do u want removed ,apart form this all the changes are completed

Comment (rithviknishad) [reply package.json:1]: All packages that you've added in this branch.

**Comment by nikhilagastya** [comment]
@rithviknishad can you check the PR for this issue.

File: src/Components/Notifications/NoticeBoard.tsx [hunk 4]
```diff
@@ -55,13 +57,28 @@ export const NoticeBoard: any = () => {
...
+
+        <div className="px-6 pt-4 pb-2">
+          <button
+            className=" mx-28 bg-primary-500 text-white hover:bg-primary-700 text-white font-bold py-2 px-4 rounded-full"
+            onClick={() => {
+              navigate("/");
+            }}
+          >
+            Go Home
+          </button>
```
Comment (rithviknishad) [inline src/Components/Notifications/NoticeBoard.tsx]: Use `ButtonV2` with its `link` prop instead. This is to keep designs consistent.

File: package-lock.json
Comment (rithviknishad) [inline package-lock.json:1]: Could you please discard changes in your `package-lock.json`?

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Notifications/NoticeBoard.tsx [hunk 5]
```diff
@@ -6,6 +6,8 @@ import { Card, CardContent } from "@material-ui/core";
 import Loading from "../Common/Loading";
 import { formatDate } from "../../Utils/utils";
 import { useTranslation } from "react-i18next";
+import bell from "../../../public/images/Notificationbell.png";
```
Comment (rithviknishad) [inline src/Components/Notifications/NoticeBoard.tsx]:
```suggestion
```

File: src/Components/Notifications/NoticeBoard.tsx [hunk 6]
```diff
@@ -55,13 +57,28 @@ export const NoticeBoard: any = () => {
     );
   } else {
     notices = (
-      <Card key="no-notice" className="my-4 rounded-lg">
-        <CardContent>
-          <div className="text-xl text-center semibold">
-            {t("no_notices_for_you")}
+      <div className=" m-auto max-w-sm rounded overflow-hidden shadow-lg ">
+        <img className="w-full" src={bell} alt="Sunset in the mountains"></img>
```
Comment (rithviknishad) [inline src/Components/Notifications/NoticeBoard.tsx]: Use the actual link to the `img` in the public dir. instead.
```suggestion
        <img className="w-full" src="/images/Notificationbell.png" alt="Sunset in the mountains" />
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by nikhilagastya** [comment]
@rithviknishad could you check if **package-lock.json** is fine now ?

File: src/Components/Notifications/NoticeBoard.tsx [hunk 7]
```diff
@@ -55,13 +57,30 @@ export const NoticeBoard: any = () => {
...
-        </CardContent>
-      </Card>
+          <p className="text-gray-700 text-base">
+            Did you know: We could render a fun fact about health here
+          </p>
+        </div>
+
+        <div className="px-6 pt-4 pb-2">
+          <ButtonV2
+            className=" mx-28 bg-primary-500 text-white hover:bg-primary-700 text-white font-bold py-2 px-4 rounded-full"
```
Comment (rithviknishad) [inline src/Components/Notifications/NoticeBoard.tsx]: Avoid specifying custom designs when using `ButtonV2`. This is to keep designs consistent across and easy to maintain.

```suggestion
            className="mx-28"
```

**Review by rithviknishad — APPROVED** [review]

File: src/Components/Notifications/NoticeBoard.tsx (hunk 7 above)
Comment (nikhilagastya) [reply src/Components/Notifications/NoticeBoard.tsx]: do u want me to send another PR for this @rithviknishad ?

Comment (rithviknishad) [reply src/Components/Notifications/NoticeBoard.tsx]: Nope, just letting you know for next time. I've already committed it. I've marked the PR as need for testing.

**Comment by nikhilagastya** [comment]
Hey @rithviknishad I have been looking forward to apply for C4GT and I have been working on my proposal , I had a few doubts ,so could we connect whenever you are free!
Thankyou

**Comment by rithviknishad** [comment]
You can message any of us on the slack channel.

**Comment by nikhilagastya** [comment]
can I get the link of the slack channel @rithviknishad

**Comment by nikhilagastya** [comment]
duplicate to #4973

**Pull request closed without merging**